resolwe/test_helpers/test_runner.py

```diff
@@ -82,6 +83,7 @@ def _manager_setup():
     update_observer_constants()
 
     manager.drain_messages()
+    redis_cache.clear()
```

Comment:
Doesn't this serve the same purpose as the clear in the previous commit?

Reply:
Almost (the other one is executed more often). I have removed this one.